docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3439,7 +3439,12 @@ An `ActionProfileGroup` entity update message has the following fields:
       `INVALID_ARGUMENT` if the client attempts to use it.
     * `watch` is the controller-defined 32-bit port number that the member's
       liveness depends on. At runtime, the member must be excluded from
-      selection if the watch port is down.
+      selection if the watch port is down.  See Section
+      [#action-selector-constraints] for notes on the behavior if all members in
+      a group are excluded for this reason. If `watch` is 0, then the member is
+      always included in the selection, regardless of the status of any port of
+      the device.  The value must be 0 or the port number of an existing port on
```

Comment:
That may be redundant with "controller-defined 32-bit port number" above, but do you think we should emphasize here and use "The value must be 0 or the SDN port number"?

Reply:
Added "SDN" qualifier to "port number"